plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -34,22 +34,16 @@
    * Objectify, App Engine API, and Google Cloud Endpoints.
    */
   public static final String APP_ENGINE_GROUP = "appengine";   //$NON-NLS-1$
-  
+
   /**
    * Library files for Google Client APIs for Java; specifically
    * google-api-client, oAuth, and google-http-client.
    */
   public static final String CLIENT_APIS_GROUP = "clientapis"; //$NON-NLS-1$
-  
-  /**
-   * Library files for all Java servlet applications; specifically
-   * servlet.jar and jsp-api.jar.
-   */
-  public static final String SERVLET_GROUP = "servlet"; //$NON-NLS-1$
```

Comment:
Now I think this string should match with the value in `plugin.xml`. (See below.) The field is not used in Java code though.
```xml
   <library
          id="servlet-api-2.5"
          name="Servlet API 2.5"
          group="servlet"
```